dashboard/app/models/script.rb

```diff
@@ -605,12 +605,12 @@ def summarize
       peer_reviews_to_complete.times do |x|
         levels << {
             ids: [x],
-            position: 0,
             kind: 'peer_review',
             title: '',
             url: '',
             name: I18n.t('peer_review.reviews_unavailable'),
-            icon: 'fa-lock'
+            icon: 'fa-lock',
```

Comment:
I'm not sure...I guess it depends on what we'd want to happen if we locked a level that had an icon.